packages/gator-permissions-snap/locales/en.json

```
@@ -0,0 +1,261 @@
+{
+  "locale": "en",
+  "messages": {
+    "recipientLabel": {
+      "message": "Recipient",
+      "description": "Label for the recipient field"
+    },
+    "recipientTooltip": {
+      "message": "The site requesting the permission",
+      "description": "Tooltip for the recipient field"
+    },
+    "networkLabel": {
+      "message": "Network",
+      "description": "Label for the network field"
+    },
+    "networkTooltip": {
+      "message": "The network on which the permission is being requested",
+      "description": "Tooltip for the network field"
+    },
+    "tokenLabel": {
+      "message": "Token",
+      "description": "Label for the token field"
+    },
+    "tokenTooltip": {
+      "message": "The token being requested",
+      "description": "Tooltip for the token field"
+    },
+    "reasonLabel": {
+      "message": "Reason",
+      "description": "Label for the reason field"
+    },
+    "reasonTooltip": {
+      "message": "Reason given by the recipient for requesting this permission.",
+      "description": "Tooltip for the reason field"
+    },
+    "accountLabel": {
+      "message": "Account",
+      "description": "Label for the account field"
+    },
+    "accountTooltip": {
+      "message": "The account from which the permission is being granted.",
+      "description": "Tooltip for the account field"
+    },
+    "initialAmountLabel": {
+      "message": "Initial Amount",
+      "description": "Label for the initial amount field"
+    },
+    "initialAmountTooltip": {
+      "message": "The initial amount of tokens that can be streamed.",
+      "description": "Tooltip for the initial amount field"
+    },
+    "maxAmountLabel": {
+      "message": "Max Amount",
+      "description": "Label for the max amount field"
+    },
+    "maxAmountTooltip": {
+      "message": "The maximum amount of tokens that can be streamed.",
+      "description": "Tooltip for the max amount field"
+    },
+    "startTimeLabel": {
+      "message": "Start Time",
+      "description": "Label for the start time field"
+    },
+    "startTimeTooltip": {
+      "message": "The time at which the first period begins(mm/dd/yyyy hh:mm:ss).",
+      "description": "Tooltip for the start time field for periodic permissions"
+    },
+    "streamStartTimeTooltip": {
+      "message": "The start time of the stream(mm/dd/yyyy hh:mm:ss).",
+      "description": "Tooltip for the start time field for stream permissions"
+    },
+    "expiryLabel": {
+      "message": "Expiry",
+      "description": "Label for the expiry field"
+    },
+    "expiryTooltip": {
+      "message": "The expiry date of the permission(mm/dd/yyyy hh:mm:ss).",
```

Missing spaces in tooltip messages
```suggestion
      "message": "The expiry date of the permission (mm/dd/yyyy hh:mm:ss).",
```